index.js

```diff
@@ -16,3 +19,26 @@ mobileLinks.forEach((link) => {
     mobileMenu.classList.toggle('active');
   });
 });
+
+form.addEventListener('submit', (e) => {
+  if (email.value !== email.value.toLowerCase()) {
+    e.preventDefault();
+    if (!email.validity) {
+      email.setCustomValidity('Email must be in lowercase');
+      form.appendChild(errorContainer);
+      errorContainer.textContent = 'Email must be in lowercase';
+      errorContainer.style.color = '#fff';
+      errorContainer.style.fontSize = '0.8rem';
+      errorContainer.style.padding = '1rem';
+      errorContainer.style.backgroundColor = 'red';
+      errorContainer.style.borderRadius = '0.5rem';
+    }
+  }
+});
+
+email.addEventListener('input', () => {
+  if (email.validity) {
+    email.setCustomValidity('');
+    form.removeChild(errorContainer);
```

Comment:
Kindly remove this line please 🙏🏽 as it is throwing an error, you also do not need to remove the `errorContainer` on every single input, if you want the error to disappear you can use aa `setTimout` in the function above. 😉

Reply:
Hi, @didierganthier. I appreciate your review. I am trying to understand your recommendations, but I, unfortunately, cannot. Kindly assist.

Reply:
Hello @10-menachi , I apologize for the delay, I was sleeping. Do you still need explanations?

Reply:
Yes, I do. Thanks.

Reply:
The way the `input` event works is that whenever a new value is entered in the input box, it triggers that event. I was just mentioning that I'm not sure it's necessary to remove the `errorContainer` every time a new character is written in the input. It throws an error at first also, because the `errorContainer` doesn't exist yet, therefore trying to remove it always gonna throw an error. It doesn't exist because you only add it later if the email is not valid.

Reply:
So I should only add the event listener on the form?

Reply:
My review wasn't about the event listener, neither was my comment. Read them carefully, please.

Reply:
Ok. Thanks